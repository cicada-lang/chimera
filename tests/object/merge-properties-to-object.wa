find q {
  q = { name: "Xie Yuheng" }
  q = { country: "China" }
}

find q {
  q = { name: "Xie Yuheng" }
  p = { name: "Xie Yuheng" }
  p = q
  p = { country: "China" }
}
