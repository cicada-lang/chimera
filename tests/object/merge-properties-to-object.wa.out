[ { "name": "Xie Yuheng", "country": "China" } ]
[ { "name": "Xie Yuheng", "country": "China" } ]
